### PR TITLE
Create pele.json

### PR DESCRIPTION
**Pull request by Pele12324**
## Requirements
Unless explicitly specified otherwise by a **maintainer** or in the requirement description, your domain must pass **ALL** the indicated requirements above.

Please note that we reserve the rights not to accept any domain at our own discretion.

- [ ] The file is in the `domains` folder and is in the JSON format.
- [ ] The file's name is all lowercased and alphanumeric. 
- [ ] You have completed your website. 
- [ ] The website is reachable.  
- [ ] The CNAME record doesn't contain `https://` or `/`.  
- [ ] There is sufficient information at the `owner` field.  

## Website Link/Preview

http://38.242.242.224/

### CODE REVIEW

File: domains/pele.json [hunk 1]
```diff
@@ -0,0 +1,10 @@
+{
+  "owner": {
+    "username": "Pele",
+    "email": "pele.puric@gmail.com",
+    "discord": "483261654818226186"
+  },
+  "record": {
+    "A": "http://38.242.242.224/"
```
Comment (cupglassDEV) [inline domains/pele.json:8]: A can only contain IP, not HTTP links
```suggestion
    "A": "38.242.242.224"
```
https://is-a.dev/docs/domain-structure

**Review by ghost — COMMENTED** [review]
You have one issue, just fix those A record problem

And you need to provide the site preview also

File: domains/pele.json (hunk 1 above)
Comment (cupglassDEV) [reply domains/pele.json:8]: Also provide a site preview, the maintainer prob ask you about that
image

Comment (Stef-00012) [reply domains/pele.json:8]: this, btw if he wants to use links, he can use URL records

**Review by Stef-DP — CHANGES_REQUESTED** [review]

File: domains/pele.json [hunk 2]
```diff
@@ -0,0 +1,10 @@
+{
+  "owner": {
+    "username": "Pele",
```
Comment (Stef-00012) [inline domains/pele.json:3]: also username should be your github username

**Review by Stef-DP — CHANGES_REQUESTED** [review]

**Comment by SkyExploreWasTaken** [comment]
gg @Pele12324 for the triple

File: domains/pele.json (hunk 1 above)
Comment (cupglassDEV) [reply domains/pele.json:8]: > this (cupglass' review), btw if he wants to use links, he can use URL records

hmm thats a good reason also

**Comment by cupglassDEV** [comment]
> gg @Pele12324 for the triple

2 Reviewers with different perspective, and a world-breaking record for triple issue labels 🔥🔥🦅

File: domains/pele.json [hunk 3]
```diff
@@ -1,10 +1,10 @@
 {
   "owner": {
-    "username": "Pele",
+    "username": "Pele12324",
     "email": "pele.puric@gmail.com",
     "discord": "483261654818226186"
   },
   "record": {
-    "A": "http://38.242.242.224/"
+    "A": "38.242.242.224"
```
Comment (cupglassDEV) [inline domains/pele.json:8]: A needs to be array-ed, like i talked to you before
```suggestion
    "A": ["38.242.242.224"]
```

**Review by ghost — COMMENTED** [review]
only one issue remaining

**Review by Pele12324 — COMMENTED** [review]
I hope this is all good, everything is how it should be now i have made the website accessible i will also make it better later but this is just for now so i can get reverse proxy and everything working.
Also here is site preview: http://38.242.242.224/

**Comment by cupglassDEV** [comment]
All good. ill ping the maintainer then, to merge @Stef-00012

**Review by Stef-DP — APPROVED** [review]

**Pull request merged**